src/app/auth/duck/reducers.ts

```diff
@@ -1,5 +1,6 @@
 import { createReducer } from 'reduxsauce';
 import { Types } from './actions';
+import { ErrorTypes } from './errorActions';
```

Comment:
Another issue with breaking this into its own file and changing the export name is you're breaking compatibility with our existing code. We should seek to maintain API compatibility with this change.